packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -241,6 +241,60 @@ void main() {
       expect(findStatic(), findsOneWidget);
     });
 
+    testWidgets('_DecoyChild preserves the child color', (WidgetTester tester) async {
+      final Widget child = getChild();
+      await tester.pumpWidget(CupertinoApp(
+        home: CupertinoPageScaffold(
+            backgroundColor: CupertinoColors.black,
+            child: MediaQuery(
+                data: const MediaQueryData(size: Size(800, 600)),
+                child: Center(
+                  child: CupertinoContextMenu(
+                      actions: const <CupertinoContextMenuAction>[
+                        CupertinoContextMenuAction(
+                          child: Text('CupertinoContextMenuAction'),
+                        ),
+                      ],
+                      child: child),
+                ))),
+      ));
+
+      // Expect no _DecoyChild to be present before the gesture
+      final Finder decoyChild = find
+          .byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_DecoyChild');
+      expect(decoyChild, findsNothing);
+
+      // Start press gesture on the child
+      final Rect childRect = tester.getRect(find.byWidget(child));
+      final TestGesture gesture = await tester.startGesture(childRect.center);
+      await tester.pump();
+
+      final Finder decoyChildDescendant = find.descendant(
+          of: decoyChild,
+          matching: find.byType(Container));
+
+      // Find the _DecoyChild by runtimeType,
+      // find the Container descendant with the BoxDecoration,
+      // then read the boxDecoration property.
+      final BoxDecoration? boxDecoration = (tester.firstWidget(decoyChildDescendant) as Container).decoration as BoxDecoration?;
+      const List<Color?> expectedColors = <Color?>[null, Color(0x00000000)];
+
+      // Expect the _DecoyChild to have the same color as the child.
+      final bool decoyChildColorMatchesAny = expectedColors
+          .any((Color? testColor) => testColor == boxDecoration?.color);
+
+      expect(decoyChildColorMatchesAny, isTrue);
```

Comment:
Nit: Could you use `contains` or something here instead of doing the `.any`? [Matcher docs](https://main-api.flutter.dev/flutter/package-matcher_matcher/package-matcher_matcher-library.html).

Reply:
Fixed in 7e80b4f52ec920b301fec11d8ea2371facecec1c